front-end/src/renderer/components/Contacts/ContactDetails.vue

```diff
@@ -151,19 +156,29 @@ watch(
       "
       class="d-flex gap-3"
     >
-      <AppButton
-        v-if="contact.user.status === 'NEW'"
-        data-testid="button-resend-email-from-contact-list"
-        class="min-w-unset"
-        color="secondary"
-        @click="handleResend"
-        >Resend email</AppButton
-      >
+      <template v-if="contact.user.status === 'NEW'">
+        <AppButton
+          data-testid="button-resend-email-from-contact-list"
+          class="min-w-unset"
+          color="secondary"
+          @click="handleResend"
+          >Resend email</AppButton
+        >
+      </template>
+      <template v-if="!contact.user.admin">
+        <AppButton
+          data-testid="button-elevate-to-admin-from-contact-list"
+          class="min-w-unset"
+          color="secondary"
+          @click="$emit('elevate-to-admin')"
+          >Elevate to Admin</AppButton
```

Comment:
Elevate makes sense, and is what I suggested. But let's change it for the UI. Let's make it 'Assign Admin', and see how that looks. No need to change the routes, or function names, etc.